resources/sdk/purecloudpython/templates/README.mustache

```diff
@@ -111,6 +111,82 @@ PureCloudPlatformClientV2.configuration.proxy_password = 'YOUR_PROXY_PASSWORD'
 
 The Python SDK uses `urllib3.ProxyManager` to make requests when `proxy` is given.
 
+#### SDK Logging
+
+Logging of API requests and responses can be controlled by a number of parameters on the `configuration`'s `logging` instance.
+
+`PureCloudPlatformClientV2.logger.LogLevel` values:
+1. LTrace (HTTP Method, URL, Request Body, HTTP Status Code, Request Headers, Response Headers)
+2. LDebug (HTTP Method, URL, Request Body, HTTP Status Code, Request Headers)
+3. LError (HTTP Method, URL, Request Body, Response Body, HTTP Status Code, Request Headers, Response Headers)
+4. LNone - default
+
+`PureCloudPlatformClientV2.logger.LogFormat` values:
+1. JSON
+2. TEXT - default
+
+By default, the request and response bodies are not logged because these can contain PII. Be mindful of this data if choosing to log it.  
+To log to a file, provide a file path to the the `PureCloudPlatformClientV2.configuration.logger.log_file_path` property. SDK users are responsible for the rotation of the log file.
+
+Example logging configuration:
+```python
+PureCloudPlatformClientV2.configuration.logger.log_level = PureCloudPlatformClientV2.logger.LogLevel.LError
```

Comment:
On the right side, it should be `PureCloudPlatformClientV2.configuration.logger.LogLevel.LError`

Reply:
I've just double checked this; it works like this because `configuration` holds an instance of the logger but to access the actual logger class you need to use `PureCloudPlatformClientV2.logger`.